packages/TelegramClient-Core.package/TCCUsers.class/instance/requestUserFor..st

```diff
@@ -0,0 +1,8 @@
+utility
+requestUserFor: aUserId
+
+	self core registerCallback:
+		[:userEvent | (self at: aUserId) resolveWith: (TCCUser newFrom: userEvent)]
+		with: (aUserId asString).
+
+	self core requestUser: aUserId with: aUserId
```

Comment:
Imo, the formatting is a bit strage here. My suggestion would be something like this:
```
requestUserFor: aUserId

	self core
                registerCallback: [:userEvent | (self at: aUserId) resolveWith: (TCCUser newFrom: userEvent)]
		with: (aUserId asString).

	self core requestUser: aUserId with: aUserId
```
or this:
```
requestUserFor: aUserId

	self core registerCallback:
                        [:userEvent | (self at: aUserId) resolveWith: (TCCUser newFrom: userEvent)]
		with: (aUserId asString).

	self core requestUser: aUserId with: aUserId
```
(the latter preferred)

Reply:
ok, calls to `self core` are on their own lines now, the block is indented once more